src/function_pool.cpp

```diff
@@ -741,7 +741,7 @@ namespace Function_Pool
     {
         Image_Function::ParameterValidation( in1, in2 );
 
-        return IsEqual( in1, 0, 0, in2, 0, 0, in1.width(), in1.height() );
+        return Function_Pool::IsEqual( in1, 0, 0, in2, 0, 0, in1.width(), in1.height() );
```

Comment:
Why do we need these changes?

Reply:
That's because we have such problems (https://travis-ci.org/ihhub/penguinV/jobs/599022738):

`error: call of overloaded ‘IsEqual(const Image&, int, int, const Image&, int, int, uint32_t, uint32_t)’ is ambiguous`

Before renaming of `PenguinV_Image` namespace, there were 3 namespaces: `PenguinV_Image`,` Function_Pool` (which had functions like `IsEqual()`) and `penguinV` (which have the same functions too). Both of `Function_Pool` and `penguinV` included only definitions from `PenguinV_Image`.

After renaming all of the names from `PenguinV_Image` is present in `penguinV`. And `Function_Pool` is using all these names now. And there are several conflicts with several functions with equal prototypes.

Reply:
Ah, okay. I'll check at home why it's like this.

Reply:
It happens because of `#include "penguinv/penguinv.h"` file inclusion.